### ISSUE TITLE
[Bookworm] tacacs/test_ro_user.py::test_ro_user_banned_by_sudoers_command failing due to unexpected sudo prompt

### ISSUE THREAD

**Issue by saiarcot895** [sonic-net/sonic-buildimage#16850]
#### Description

On Bookworm, after the tacplus library compilation and system configuration has been fixed (#16831), `tacacs/test_ro_user.py::test_ro_user_banned_by_sudoers_command` fails. It looks like it's expecting the message "Make sure your account has RW permission to current device", but instead there's a sudo password prompt.

Test results: https://elastictest.org/scheduler/testplan/652625b1f773ca326e14c0fd?testcase=tacacs%2Ftest_ro_user.py&type=console

#### Steps to reproduce the issue:
1.
2.
3.

#### Describe the results you expected:

Snippet from the log:

```
        for command in commands:
            if does_command_exist(localhost, dutip, tacacs_creds['tacacs_ro_user'],
                                  tacacs_creds['tacacs_ro_user_passwd'], command):
                for subcommand in commands[command]:
                    allowed = ssh_remote_allow_run(localhost, dutip, tacacs_creds['tacacs_ro_user'],
                                                   tacacs_creds['tacacs_ro_user_passwd'], subcommand)
>                   pytest_assert(not allowed, "command '{}' not block by sudoers".format(subcommand))
E                   Failed: command 'sudo cat /etc/hosts' not block by sudoers

allowed    = True
check_tacacs = None
command    = 'cat'
commands   = {'cat': ['sudo cat /etc/hosts', 'sudo cat /var/log/syslog /etc/hosts', 'sudo cat /var/log/syslog.1 /etc/hosts']}
duthost    = <MultiAsicSonicHost vlab-08>
duthosts   = [<MultiAsicSonicHost vlab-08>]
dutip      = '10.250.0.112'
enum_rand_one_per_hwsku_hostname = 'vlab-08'
localhost  = <tests.common.devices.local.Localhost object at 0x7f05de49f370>
subcommand = 'sudo cat /etc/hosts'
tacacs_creds = {'local_user': 'test_louser', 'local_user_passwd': '123456', 'tacacs_authorization_user': 'test_auuser', 'tacacs_authorization_user_passwd': '123456', ...}
```

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by saiarcot895** [comment]
Debian [changed the defaults of sudo](https://salsa.debian.org/sudo-team/sudo/-/commit/871f54c80b76ce8d3590d5934702a1ed3cf77903 to match Ubuntu, and default to never lecturing the user (so /etc/sudoers.lecture is never used). Modifying the sudoers file to the old default of lecturing once works, and appears to match current behavior.

**Comment by saiarcot895** [comment]
Fixed in a7bfb55
